application/app_files/jenkins-job.default.xml.twig

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+    <actions/>
+    <description>{% block description %}{% if job.description is not empty %}{{ job.description }}{% endif %}{% endblock description %}</description>
+    {% block logRotator %}
+    <logRotator>
+        <daysToKeep>{{ job.payload.logrotate_days }}</daysToKeep>
+        <numToKeep>100</numToKeep>
+        <artifactDaysToKeep>{{ job.payload.logrotate_days }}</artifactDaysToKeep>
+        <artifactNumToKeep>-1</artifactNumToKeep>
+    </logRotator>
+    {% endblock logRotator %}
+    <keepDependencies>false</keepDependencies>
+    <properties/>
+    <scm class='hudson.scm.NullSCM'/>
+    <canRoam>true</canRoam>
+    <disabled>{% block disable %}{{ not job.enable }}{% endblock disable %}</disabled>
+    <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+    <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+    {% block triggers %}
+    <triggers class='vector'>
+        <hudson.triggers.TimerTrigger>
+            <spec>{{ job.repeat_pattern }}</spec>
+        </hudson.triggers.TimerTrigger>
+    </triggers>
+    {% endblock triggers %}
+    <concurrentBuild>false</concurrentBuild>
+    <builders>
+        <org.jvnet.hudson.plugins.SSHBuilder plugin='ssh@2.6.1'>
+          <siteName>www-data@@appHost@:222</siteName>";
+            <command>
+                {%- block setup %}
+                    [ -z "${PHP_BIN}" ] &amp;&amp; export PHP_BIN=php
+                    source /versions/vars
+                    export APPLICATION_ENV={{ getenv('APPLICATION_ENV') }}
+                    export APPLICATION_STORE={{ job.store }}
+                    cd {{ working_dir }}
```

Comment:
`cd {{ working_dir }}` should be deleted because the directory name will be changed with every new deploy.
or use `cd /data`

Reply:
Will change it to 'cd /data'